drivers/backup/portworx/portworx.go

```diff
@@ -134,7 +136,7 @@ func (p *portworx) Init(schedulerDriverName string, nodeDriverName string, volum
 }
 
 func (p *portworx) constructURL(ip string) string {
-	return fmt.Sprintf("%s:%d", ip, defaultPxbServicePort)
+	return net.JoinHostPort(ip, strconv.Itoa(int(defaultPxbServicePort)))
```

Comment:
Don't need to type cast int(defaultPxbServicePort) since its already an int...

Reply:
i think it's int32, which `strconv.Itoa` complain about. that's why i recaste it to int.

Reply:
I thought it was const?